.github/workflows/ci-terraform.yml

```
@@ -0,0 +1,20 @@
+on:
+  push:
+    paths:
+      - 'infra/terraform/**'
+  pull_request:
+    paths:
+      - 'infra/terraform/**'
+
+jobs:
+  terraform:
+    name: Terraform fmt/validate/plan
+    uses: ./.github/workflows/../ci/reusable/terraform.yml
```

actionlint (1.7.7)

could not read reusable workflow file for "./.github/workflows/../ci/reusable/terraform.yml": open /home/jailuser/git/.github/ci/reusable/terraform.yml: no such file or directory

(workflow-call)